src/lib/data/buyables/canifisClothes.ts

```diff
@@ -0,0 +1,75 @@
+import { resolveNameBank } from '../../util';
+import { Buyable } from './buyables';
+
+export const canifisClothes: Buyable[] = [
```

Comment:
these should have a qp requirement since getting to canifis requires a quest, something like 20 or so since its not a hard one.